lib/timeline/component/Group.js

```diff
@@ -486,7 +484,7 @@ Group.prototype._updateSubgroupsSizes = function () {
   if (me.subgroups) {
     for (var subgroup in me.subgroups) {
       var newStart = me.subgroups[subgroup].items[0].data.start;
-      var newEnd = me.subgroups[subgroup].items[0].data.end;
+      var newEnd = me.subgroups[subgroup].items[0].data.end - 1;
 
       me.subgroups[subgroup].items.forEach(function(item) {
         if (new Date(item.data.start) < new Date(newStart)) { 
```

Comment:
Ah right, string dates. I haven't thought that far. OK then the comparisons make sense. However, see new points.